Sources/StructuralCore/Structural.swift

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import Foundation
```

Comment:
What is `Foundation` being used for?

Reply:
It was used to experiment with `NSExpression`-based extraction of name from KeyPath, but it's not needed any more.

Reply:
Dropped this import in e52bb75